pkg/README.md

```diff
@@ -1,45 +1,46 @@
 
 ## Global state
 
-Global state makes testing and debugging software harder and its something we want to avoid when possible.
-Unfortunately, there is quite a lot of global state in Grafana. The way we want to migrate away from this
-is to use the `inject` package to wire up all dependencies either in `pkg/cmd/grafana-server/main.go` or
-self registering using `registry.RegisterService` ex https://github.com/grafana/grafana/blob/master/pkg/services/cleanup/cleanup.go#L25
+Global state makes testing and debugging software harder and it's something we want to avoid when possible. Unfortunately, there is quite a lot of global state in Grafana. 
+
+We want to migrate away from this by using the `inject` package to wire up all dependencies either in `pkg/cmd/grafana-server/main.go` or self-registering using `registry.RegisterService` ex https://github.com/grafana/grafana/blob/master/pkg/services/cleanup/cleanup.go#L25.
 
-## Reduce the use of the init() function
+## Limit the use of the init() function
 
-Should only be used to register services/implementations.
+Only use the init() function to register services/implementations.
 
 ## Settings refactoring
 
-The plan is to move all settings to from package level vars in settings package to the [setting.Cfg](https://github.com/grafana/grafana/blob/df917663e6f358a076ed3daa9b199412e95c11f4/pkg/setting/setting.go#L210) struct. To access the settings services/components can inject this setting.Cfg struct.
+The plan is to move all settings to from package level vars in settings package to the [setting.Cfg](https://github.com/grafana/grafana/blob/df917663e6f358a076ed3daa9b199412e95c11f4/pkg/setting/setting.go#L210) struct. To access the settings, services and components can inject this setting.Cfg struct:
 
 [Cfg struct](https://github.com/grafana/grafana/blob/df917663e6f358a076ed3daa9b199412e95c11f4/pkg/setting/setting.go#L210)
 [Injection example](https://github.com/grafana/grafana/blob/df917663e6f358a076ed3daa9b199412e95c11f4/pkg/services/cleanup/cleanup.go#L20)
 
-## Reduce the use of Goconvey
+## Reduce the use of GoConvey
 
-We want to migrated away from using Goconvey and use stdlib testing as its the most common approuch in the GO community and we think it will make it easier for new contributors. Read more about how we want to write tests in the [ARCHITECTURE.MD](ARCHITECTURE.md#Testing) docs.
+We want to migrate away from using GoConvey. Instead, we want to use stdlib testing, because it's the most common approach in the Go community and we think it will be easier for new contributors. Read more about how we want to write tests in [ARCHITECTURE.MD](ARCHITECTURE.md#Testing).
 
-## Sqlstore refactoring
+## SQLStore refactoring
```

Comment:
SQL store might refer to either the [package name](https://github.com/grafana/grafana/blob/6809d2bb292e57efb685e1a676ed0d2b9a3635ab/pkg/services/sqlstore/sqlstore.go#L1)

... or [the identifier](https://github.com/grafana/grafana/blob/6809d2bb292e57efb685e1a676ed0d2b9a3635ab/pkg/services/sqlstore/sqlstore.go#L52).

In this case, I think we should use `SqlStore` for all occurrences.

Reply:
SQL = Structured Query Language.

ARGH. This is the sort of thing that occasionally makes software documentation tricky.

I see both those examples are in the pkg directory. Are both of them things that would or should or need to be refactored? If possible, I would like to be specific about which thing we are talking about in regards to refactoring.

Reply:
In this case, I'd love to refactor the code itself into `SQLStore` but that's arguably out of scope for this pull request.

Reply:
Since I don't know how to fix whatever I would break in the process, probably best. 

Much as I hate resorting to slashes, perhaps 'SqlStore'/'sqlstore' in the heading, and for the text:

>The `SqlStore` and `sqlstore` handlers all use the... Refactor them to SQLStore. 

Does that make sense, or did I change the meaning too much by removing "instance"?

Reply:
I looked into it, and I think this should be the correct capitalization:

```
## SqlStore refactoring

The `sqlstore` handlers all use a global xorm engine variable. Refactor them to use the `SqlStore` instance.
```

The `sqlstore` handler is the old way.
The `SqlStore` instance is the new way.

Reply:
I still need the issue number. I could not find the issue we discussed.